mongoimport/mongoimport.go

```diff
@@ -454,27 +475,6 @@ func (imp *MongoImport) ingestDocuments(readDocs chan bson.D) (retErr error) {
 	return
 }
 
-// configureSession takes in a session and modifies it with properly configured
-// settings. It does the following configurations:
-//
-// 1. Sets the session to not timeout
-// 2. Sets the write concern on the session
-// 3. Sets the session safety
-//
-// returns an error if it's unable to set the write concern
-func (imp *MongoImport) configureSession(session *mgo.Session) error {
```

Comment:
`newSessionProvider` takes care of all this stuff now right?

Reply:
Yes. The write concern parsing logic is all done in ParseOptions and then gets passed into newSessionProvider.